docs/en/Debugging-Guide.md

```diff
@@ -58,6 +59,48 @@ bucket, without the `s3://`, `s3a://`, or `s3n://` prefix.
 
 ## Usage FAQ
 
+#### Q: Why I see exceptions like "No FileSystem for scheme: alluxio"
+
+A: This error message is seen when your applications (e.g., MapReduce, Spark and etc) tried to access
```

Comment:
Using both `e.g.` and `etc` is superfluous, let write just `(e.g., MapReduce and Spark)`

tried -> try